src/mina-signer/tests/rosetta.test.ts

```diff
@@ -1,17 +1,45 @@
-import Client from '../dist/node/mina-signer/mina-signer.js';
+import Client from '../dist/node/mina-signer/MinaSigner.js';
+import { fieldFromHex, fieldToHex, signatureJsonToHex, signatureJsonFromHex, UnsignedTransaction, publicKeyToHex, signTransaction } from '../dist/node/mina-signer/src/rosetta.js';
+import { PublicKey } from '../dist/node/provable/curve-bigint.js';
+import { Field } from '../dist/node/provable/field-bigint.js';
 
 describe('Rosetta', () => {
   let client: Client;
 
+  const rosettaUnsignedTxnString = '{"randomOracleInput":"0000000333E1F14C6155B706D4EA12CF70685B8DCD3342A8B36A27CC3EB61B5871F9219E33E1F14C6155B706D4EA12CF70685B8DCD3342A8B36A27CC3EB61B5871F9219E33E1F14C6155B706D4EA12CF70685B8DCD3342A8B36A27CC3EB61B5871F9219E000002570242F000000000008000000000000000C00000007FFFFFFFC00000000000000000000000000000000000000000000000000000000000000000000E0000000000000000014D677000000000","signerInput":{"prefix":["33E1F14C6155B706D4EA12CF70685B8DCD3342A8B36A27CC3EB61B5871F9219E","33E1F14C6155B706D4EA12CF70685B8DCD3342A8B36A27CC3EB61B5871F9219E","33E1F14C6155B706D4EA12CF70685B8DCD3342A8B36A27CC3EB61B5871F9219E"],"suffix":["0000000000000007FFFFFFFC00000006000000000000000200000000001E8480","0000000003800000000000000000000000000000000000000000000000000000","00000000000000000000000000000000000000000000000001DCD65000000000"]},"payment":{"to":"B62qqQomCgjaKhayN79wWqDNsSJKFaZjrkuCp8Kcrt36ubXb14XHU2X","from":"B62qqQomCgjaKhayN79wWqDNsSJKFaZjrkuCp8Kcrt36ubXb14XHU2X","fee":"1000000","token":"wSHV2S4qX9jFsLjQo8r1BsMLH2ZRKsZx6EJd1sbozGPieEC4Jf","nonce":"1","memo":null,"amount":"1000000000","valid_until":null},"stakeDelegation":null}';
+  const rosettaUnsignedTxn: UnsignedTransaction = JSON.parse(rosettaUnsignedTxnString);
```

Comment:
nit: could this test hard-code the `UnsignedTransaction` directly instead of the string? would be more readable and type-safe